rmw_opendds_cpp/src/rmw_node.cpp

```diff
@@ -30,47 +30,39 @@ rmw_create_node(
   size_t domain_id,
   bool localhost_only)
 {
-  RCUTILS_CHECK_ARGUMENT_FOR_NULL(context, NULL);
-  RMW_CHECK_TYPE_IDENTIFIERS_MATCH(
-    init context,
-    context->implementation_identifier,
-    opendds_identifier,
-    // TODO(wjwwood): replace this with RMW_RET_INCORRECT_RMW_IMPLEMENTATION when refactored
```

Comment:
Should we leave this TODO in?

Reply:
I don't think it's necessary, since when the interface changes to return the object in an out parameter, the function return code here is obvious.